lib/internal/bootstrap_node.js

```diff
@@ -261,7 +261,8 @@
       enumerable: true,
       get: function() {
         if (!console) {
-          console = installInspectorConsole(originalConsole);
+          console = originalConsole ? installInspectorConsole(originalConsole) :
```

Comment:
this makes much more sense 👍 
Is this the right check? i.e. if `inspector` is "on" `global.console` will be false?